BookShelf/Editions/Counter.swift

```diff
@@ -0,0 +1,14 @@
+
+
+struct Counter {
+    static var countId = 0
+    
+    static func getCountId() -> Int {
```

Comment:
```suggestion
    static func getCountID() -> Int {
```

Просто кодстайл, не более)